### ISSUE TITLE
How to know "OnNeuronRatio" in TF2

### ISSUE THREAD

**Issue by workdd** [aws-neuron/aws-neuron-sdk#621]
Hello!
I was able to check the OnNeuronRatio value when compiling the model in TF1. But I can't check it in TF2 Can I check OnNeuronRatio in TF too? I want to put this metric to good use.

Here is code of TF1 for checking OnNeuronRatio
```
compiled_res = tfn.saved_model.compile(saved_model_dir, compiled_model_dir)
    print(f'Compile time: {time.time() - start_time}')
    
    compile_success = False
    perc_on_inf = compiled_res['OnNeuronRatio'] * 100
    if perc_on_inf > 50:
        compile_success = True
```

Thank you

**Comment by jluntamazon** [comment]
Hi @workdd,
Currently the TensorFlow 2.0 `trace` function does not return the compilation statistics. The logs will show the operators that are unsupported but not the complete ratio that you are using from the `compile` function . For example:
```
There are 3 ops of 2 different types in the graph that are not compiled by neuron-cc: Placeholder, NoOp, (For more information see https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/neuron-cc-ops/neuron-cc-ops-tensorflow.html).
```

If you do need a programmatic option, let us know since this is something we could look to support in a future release.

**Comment by jeffhataws** [comment]
Hello @workdd, we are working on a fix for this issue. Thanks for your patience.

**Comment by workdd** [comment]
I'm looking forward to fix it ! Thank you for your response :)

**Comment by mrnikwaws** [comment]
Hi @workdd,

The 2.9.0 release of the neuron SDK (https://awsdocs-neuron.readthedocs-hosted.com/en/latest/release-notes/index.html) added support for this. If the OnNeuronRatio is less than 50% it will be logged.

You can also use the new analyze_model call (https://awsdocs-neuron.readthedocs-hosted.com/en/latest/frameworks/tensorflow/tensorflow-neuron/api-tfn-analyze-model-api.html) to get this information for a TF2 model.

Since the issue seems to be resolved I am going to close this ticket.  Please open a new ticket if there is a problem with the feature, or it doesn't meet your needs.
